docs/installation/introduction.rst

```diff
@@ -84,128 +82,133 @@ offering the finest control over the entire platform:
 
 Compact Architectures
 """""""""""""""""""""
-While not being focused on having the smallest compute and memory footprints,
-MetalK8s can provide a fully functional single node "cluster". The Bootstrap
-node can be configured to also allow running applications next to all the other
-services required (see :ref:`the section about taints<node-taints>` below).
 
-A single node cluster does not provide any form of resilience to machine or
-site failure, which is why the recommended most compact architecture to use in
-production includes three machines:
+Although its design is not focused on having the smallest compute and memory
+footprints, MetalK8s can provide a fully functional single-node "cluster". The
+bootstrap node can be configured to also allow running applications next to all
+other required services (see :ref:`the section about taints<node-taints>`
+below).
+
+Because a single-node cluster has no resilience to machine or site failure, a
+three-machine cluster is the most compact recommended production
+architecture. This architecture includes:
 
 - Two machines running control plane services alongside infra and workload
   applications
-- One machine running Bootstrap services in addition to all the other services
+- One machine running bootstrap services and all other services
 
 .. image:: img/compact-arch.png
    :width: 100%
 
 .. note::
 
-   Sizing of such compact clusters needs to account for the expected load,
-   and the exact impact of colocating an application with MetalK8s services
-   needs to be evaluated by said application's provider.
-
+   Sizing for such compact clusters must account for the expected load. The
+   exact impact of colocating an application with MetalK8s services must be
+   evaluated by that application's provider.
 
 Variations
 """"""""""
-It is possible to customize the chosen architecture using combinations of
-:ref:`roles<node-roles>` and :ref:`taints<node-taints>`, which are described
-below, to adapt to the available infrastructure.
 
-As a general recommendation, it is easier to monitor and operate well-isolated
-groups of machines in the cluster, where hardware issues would only impact one
-group of services.
+You can customize your architecture using combinations of
```

Comment:
I thought our style guide was to never directly address the reader, hence omitting any use of "You". Could we have a clear document to answer this kind of question, so we don't do back-n-forth PRs to change this kind of details?
cc @lucieleonard

Reply:
I appreciate your attention to the rules, but I have never followed that rule, and I am pushing hard to relegate it to history: it does not make for good writing or reading, hides agency, and makes for long, tortured sentences. 

Let me talk through this with Lucie and I'll revise the rules. Until then, please let this stand.